doc/classes/Label.xml

```diff
@@ -127,6 +127,7 @@
 		<theme_item name="outline_size" data_type="constant" type="int" default="0">
 			Text outline size.
 			[b]Note:[/b] If using a font with [member FontFile.multichannel_signed_distance_field] enabled, its [member FontFile.msdf_pixel_range] must be set to at least [i]twice[/i] the value of [theme_item outline_size] for outline rendering to look correct. Otherwise, the outline may appear to be cut off earlier than intended.
+			[b]Note:[/b] Using a large radius is not recommended.
```

Comment:
This isn't called a radius elsewhere here, so I'd instead do:
```suggestion
			[b]Note:[/b] Using a large value is not recommended.
```
But "large" here is very unclear, it could mean really anything, is it 10, or 100, or 1000?

So needs to be more specific